### PR TITLE
Add avahi zeroconf support

### PR DESCRIPTION
**Pull request by mrvovanness**
There is option to use avahi for automatic ip and DNS management described in  https://www.tinc-vpn.org/examples/zeroconf-ip-and-dns/. This merge request allow use this configuration by adding attrubute `avahi_zeroconf_enabled`.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -16,8 +16,7 @@ verifier:
   name: inspec
 
 platforms:
-  - name: debian-8.6
-  - name: debian-9.0
+  - name: ubuntu-16.04
```
Comment (EugenMayer) [inline .kitchen.yml]: please debian only, do not change this

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Tinc Virtual Private Network'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '0.1.9'
+version           '0.1.10'
```
Comment (EugenMayer) [inline metadata.rb]: please revert that, i release that when it is a bundle, thanks

**Review by EugenMayer — COMMENTED** [review]
great, thanks for the contribution!

If you could adjust the commented lines we are good to go!

**Comment by EugenMayer** [comment]
Great stuff, @mrvovanness are you ready - is this release able? ( can i merge)

**Comment by mrvovanness** [comment]
@EugenMayer Yes. it ready

**Comment by EugenMayer** [comment]
Thank you a lot for the contribution!

**Pull request merged**